### PR TITLE
Break down JDBC module

### PR DESCRIPTION
**Pull request by mmoayyed**
Related to #1192

### CODE REVIEW

File: cas-server-support-jpa-service-registry/.cvsignore [hunk 1]
```diff
@@ -0,0 +1 @@
+target
```
Comment (leleuj) [inline cas-server-support-jpa-service-registry/.cvsignore]: I don't think we need to keep this file: `.cvsignore`, don't we?

File: cas-server-support-jpa-service-registry/src/test/resources/jpaSpringContext.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+       http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx.xsd
+       http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop.xsd
+       http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+       http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd
+       http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+
+
+
+  <context:property-placeholder properties-ref="casProperties"/>
+  <util:properties id="casProperties"/>
```
Comment (leleuj) [inline cas-server-support-jpa-service-registry/src/test/resources/jpaSpringContext.xml]: Strange to have two lines on properties...

File: cas-server-support-jpa-service-registry/src/main/resources/META-INF/spring/jpa-svc-reg-context.xml [hunk 3]
```diff
@@ -35,6 +35,9 @@
        http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd
        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
 
+    <context:component-scan base-package="org.jasig.cas" />
```
Comment (leleuj) [inline cas-server-support-jpa-service-registry/src/main/resources/META-INF/spring/jpa-svc-reg-context.xml]: This package are already scanned: https://github.com/Jasig/cas/blob/master/cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/applicationContext.xml#L42

File: cas-server-webapp/src/main/webapp/WEB-INF/cas.properties [hunk 4]
```diff
@@ -34,23 +34,46 @@ server.prefix=${server.name}/cas
...
-# database.pool.minSize=6
-# database.pool.maxSize=18
-# database.pool.maxWait=10000
-# database.pool.maxIdleTime=120
-# database.pool.acquireIncrement=6
-# database.pool.idleConnectionTestPeriod=30
-# database.pool.connectionHealthQuery=select 1
-# database.pool.acquireRetryAttempts=5
-# database.pool.acquireRetryDelay=2000
+# JPA Ticket Registry Database Configuration
```
Comment (leleuj) [inline cas-server-webapp/src/main/webapp/WEB-INF/cas.properties]: And our `cas.properties` is getting bigger again ;-)

File: cas-server-support-jpa-ticket-registry/.cvsignore [hunk 5]
```diff
@@ -0,0 +1 @@
+target
```
Comment (leleuj) [inline cas-server-support-jpa-ticket-registry/.cvsignore]: One useless file.

**Comment by leleuj** [comment]
Looks good to me, a few comments here and there.

File: cas-server-support-jpa-service-registry/src/test/resources/jpaSpringContext.xml (hunk 2 above)
Comment (mmoayyed) [inline cas-server-support-jpa-service-registry/src/test/resources/jpaSpringContext.xml]: One is used to scan the context to replace properties. Once actually makes cas properties available to the context as Properties object.

File: cas-server-support-jpa-service-registry/src/main/resources/META-INF/spring/jpa-svc-reg-context.xml (hunk 3 above)
Comment (mmoayyed) [inline cas-server-support-jpa-service-registry/src/main/resources/META-INF/spring/jpa-svc-reg-context.xml]: Valid point. I think most modules have this definition. I will keep this here for now, but in a separate issue, will go through to remove duplicate scans.

**Comment by mmoayyed** [comment]
Removed cvsignore files and restested both modules. Works just as intended, and it's very simpler and easier to manage now.

**Comment by mmoayyed** [comment]
I'll proceed with LDAP next.

**Pull request merged**